.circleci/config.yml

```diff
@@ -33,25 +46,28 @@ references:
        git checkout stable
        cd ..
 
-  build_push_docker: &build_push_docker
+  install_jsonnet: &install_jsonnet
+    run:
+      name: Install jsonnet
+      command: |
+        go get github.com/google/go-jsonnet/cmd/jsonnet
+
+  update_jsonnet: &update_jsonnet
     run:
-      name: Build and push Docker image
-      environment:
-        - PYTHON_VER: 3.7
+      name: Update jsonnet
       command: |
-        gcloud --quiet auth configure-docker
-        #cd dockers/tpu-tests
-        docker build --tag "$GCR_IMAGE_PATH:$CIRCLE_WORKFLOW_JOB_ID" -f ./dockers/tpu-tests/Dockerfile --build-arg "PYTHON_VERSION=$PYTHON_VER" --build-arg "PYTORCH_VERSION=$XLA_VER" .
-        docker push "$GCR_IMAGE_PATH:$CIRCLE_WORKFLOW_JOB_ID"
+       export PR_NUMBER=$(git ls-remote origin "pull/*/head" | grep -F -f <(git rev-parse HEAD) | awk -F'/' '{print $3}')
+       export SHA=$(git rev-parse --short HEAD)
+       python -c "fname = 'dockers/tpu-tests/tpu_test_cases.jsonnet' ; fff = open(fname).read().replace('{PYTORCH_VERSION}', '$XLA_VER')
```

Comment:
what's `fff` supposed to mean?

Reply:
@carmocca It's just the variable/placeholder for jsonnet data.

Reply:
but why `fff` and not `cases` or `data`

Reply:
Updated!